test/lint/lint-includes.sh

```diff
@@ -108,4 +111,6 @@ if [[ ${QUOTE_SYNTAX_INCLUDES} != "" ]]; then
     EXIT_CODE=1
 fi
 
+cd $OLDPWD || exit 1
```

Comment:
nit, unnecessary, first `cd` happens in the sub shell - calling shell isn't affected.

Reply:
Ohh, yes, you're right!